src/tiled/addpropertydialog.h

```diff
@@ -0,0 +1,32 @@
+#ifndef ADDPROPERTYDIALOG_H
+#define ADDPROPERTYDIALOG_H
+
+#include <QDialog>
+
+namespace Ui {
+class AddPropertyDialog;
+}
+
+class AddPropertyDialog : public QDialog
+{
+    Q_OBJECT
+
+public:
+    explicit AddPropertyDialog(QWidget *parent = 0);
+    ~AddPropertyDialog();
+
+
+    QString getPropertyName();
+    QVariant::Type getPropertyType();
```

Comment:
Please follow the existing coding style, in that getters do not have "get" in their name.